LICENSE

```diff
@@ -321,12 +321,12 @@ For the jwt-go component:
 
 For the pq component:
 @traffic_ops/experimental/traffic_ops_auth/vendor/github.com/lib/pq/*
-@traffic_ops/testing/api/vendor/github.com/lib/pq/*
+@traffic_ops/vendor/github.com/lib/pq/*
 ./traffic_ops/experimental/traffic_ops_auth/vendor/github.com/lib/pq/LICENSE.md
 
 and also:
-@traffic_ops/traffic_ops_golang/vendor/github.com/lib/pq/*
-./traffic_ops/traffic_ops_golang/vendor/github.com/lib/pq/LICENSE.md
+@traffic_ops/vendor/github.com/lib/pq/*
```

Comment:
Should this be removed, since it's the same line as above now?

Reply:
correct. Will update the PR.